spec/services/credits/applied_coupon_service_spec.rb

```diff
@@ -131,7 +131,14 @@
       let(:coupon) { create(:coupon, frequency: 'recurring', frequency_duration: 3) }
 
       let(:applied_coupon) do
-        create(:applied_coupon, coupon: coupon, frequency: 'recurring', frequency_duration: 3, amount_cents: 12)
+        create(
+          :applied_coupon,
+          coupon: coupon,
+          frequency: 'recurring',
+          frequency_duration: 3,
+          frequency_duration_remaining: 3,
+          amount_cents: 12
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)